src/Libraries/RevitNodes/AnalysisDisplay/AbstractAnalysisDisplay.cs

```diff
@@ -277,7 +271,9 @@ protected void SetElementAndPrimitiveIdsForTrace(SpatialFieldManager manager, Li
                 SpatialFieldManagerID = manager.Id.Value,
                 PrimitiveIds = primitiveIds
             };
-            ElementBinder.SetRawDataForTrace(idPair);
+
+            var serializedTraceData = JsonConvert.SerializeObject(idPair);
+            ElementBinder.SetRawDataForTrace(serializedTraceData);
```

Comment:
does it make more sense to have the ElementBinder internally do the serialization / deserialization?